src/Integration/Aws/DynamoDbSession/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "async-aws/dynamo-db-session",
+    "type": "library",
+    "description": "AsyncAws DynamoDB integration for PHP Sessions.",
+    "keywords": [
+        "aws",
+        "async-aws",
+        "dynamo-db",
+        "session"
+    ],
+    "license": "MIT",
+    "require": {
+        "php": "^7.2.5",
+        "async-aws/dynamo-db": "^0.2"
```

Comment:
will be `^0.3` if use `UpdateTimeToLiveInput`

Reply:
I has fixed that here already:
https://github.com/async-aws/aws/pull/617/commits/560e40b50d7cef08ee212fdb396576a668908b65